test/extensions/filters/http/wasm/test_data/test_cpp.cc

```diff
@@ -578,6 +578,20 @@ void TestRootContext::onTick() {
       logDebug("missing node metadata");
     }
     logDebug(std::string("onTick ") + value);
+
+    std::string list_value;
+    if (!getValue({"node", "metadata", "wasm_node_list_key", "0"}, &list_value)) {
+      logDebug("missing node metadata list value");
+    }
+    if (list_value != "wasm_node_get_value") {
+      logWarn("unexpected list value: " + list_value);
+    }
+    if (getValue({"node", "metadata", "wasm_node_list_key", "bad_key"}, &list_value)) {
+      logDebug("unexpected list value for a bad_key");
+    }
+    if (getValue({"node", "metadata", "wasm_node_list_key", "1"}, &list_value)) {
+      logDebug("unexpected list value outside the range");
+    }
```

Comment:
Does the test fail if the debug statements fail? Does the test actually verify any values? If not please make this test more robust so it actually checks things. Thank you!

/wait

Reply:
This is just how these tests work. All log statements must have a mock expectation. Since I didn't add those expectations, the log calls are not hit.

Reply:
Ah OK, sounds good, thanks.